src/box/lua/config/init.lua

```diff
@@ -143,6 +143,7 @@ function methods._initialize(self)
         self:_register_source(require('internal.config.source.file').new())
     end
 
+    self:_register_applier(require('internal.config.applier.lua'))
```

Comment:
It doesn't change things much, but I would add it to `first_phase_appliers` (see below) for consistency with all the other pre-box-cfg appliers.

Reply:
Done. Thank you.